applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -11,28 +11,29 @@
 
 #if (CONFIG_SW_CODEC_SBC)
 /* Frame length in samples */
-#define SBC_FRAME_LEN_SAMPLES \
-	(CONFIG_SBC_NO_OF_BLOCKS * CONFIG_SBC_NO_OF_SUBBANDS)
+#define SBC_FRAME_LEN_SAMPLES (CONFIG_SBC_NO_OF_BLOCKS * CONFIG_SBC_NO_OF_SUBBANDS)
 /* Frame rate in Hz */
 #define SBC_FRAME_RATE_HZ (CONFIG_AUDIO_SAMPLE_RATE_HZ / SBC_FRAME_LEN_SAMPLES)
 /* Frame size in bytes */
-#define SBC_ENC_MONO_FRAME_SIZE (4 + (CONFIG_SBC_NO_OF_BLOCKS * \
-	CONFIG_SBC_BITPOOL + 4 * CONFIG_SBC_NO_OF_SUBBANDS) / 8)
+#define SBC_ENC_MONO_FRAME_SIZE                                                                    \
+	(4 + (CONFIG_SBC_NO_OF_BLOCKS * CONFIG_SBC_BITPOOL + 4 * CONFIG_SBC_NO_OF_SUBBANDS) / 8)
 /* Bitrate in bits/s */
 #define SBC_MONO_BITRATE (SBC_ENC_MONO_FRAME_SIZE * 8 * SBC_FRAME_RATE_HZ)
 /* PCM frame size in bytes */
 #define PCM_NUM_BYTES_SBC_FRAME_MONO ((SBC_FRAME_LEN_SAMPLES * CONFIG_AUDIO_BIT_DEPTH_BITS) / 8)
 /* Size of PCM frames sent per BLE packet, in bytes */
-#define SBC_PCM_NUM_BYTES_MONO \
-	(PCM_NUM_BYTES_SBC_FRAME_MONO * CONFIG_SBC_NUM_FRAMES_PER_BLE_PACKET)
+#define SBC_PCM_NUM_BYTES_MONO (PCM_NUM_BYTES_SBC_FRAME_MONO * CONFIG_SBC_NUM_FRAMES_PER_BLE_PACKET)
 /* Encoded total frame size in bytes */
-#define SBC_ENC_MAX_FRAME_SIZE \
-	(SBC_ENC_MONO_FRAME_SIZE * CONFIG_SBC_NUM_FRAMES_PER_BLE_PACKET)
+#define SBC_ENC_MAX_FRAME_SIZE (SBC_ENC_MONO_FRAME_SIZE * CONFIG_SBC_NUM_FRAMES_PER_BLE_PACKET)
+#define SBC_ENC_TIME_US 3000
```

Comment:
So SBC and LC3 uses approx the same amount of time?

Reply:
No, but I didn't want to spend time on SBC when it is going to be removed